utils/slackbots/suggestion-bot/index.js

```diff
@@ -0,0 +1,123 @@
+// const AWS = require('aws-sdk');
+const axios = require('axios');
+const qs = require('qs');
+
+const signature = require('./verifySignature');
+const message = require('./message');
+
+const apiUrl = 'https://slack.com/api';
```

Comment:
I think that the channel ID should definitely remain public. Reason 1 is that technically all of our channel IDs are already public since the workspace is public. Reason 2 is that we want to have transparency about what channel the bot is "listening" to.

Reply:
Makes sense! Disregard this comment.